src/Microsoft.AspNet.Mvc.Razor/IRazorViewFactory.cs

```diff
@@ -18,6 +19,6 @@ public interface IRazorViewFactory
         /// <param name="page">The <see cref="IRazorPage"/> instance to execute.</param>
         /// <param name="isPartial">Determines if the view is to be executed as a partial.</param>
         /// <returns>A <see cref="IView"/> instance that renders the contents of the <paramref name="page"/></returns>
-        IView GetView(IRazorViewEngine viewEngine, IRazorPage page, bool isPartial);
+        IView GetView(IRazorViewEngine viewEngine, IRazorPage page, IReadOnlyList<IRazorPage> viewStarts, bool isPartial);
```

Comment:
This doesn't seem useful as an extensibility point, can we just move this code into the view engine directly?